### PR TITLE
Add some information to the API Token header

### LINKED ISSUE #7582: Wrong swagger api for AuthorizationHeaderToken  (apiKey)
image

The header should be `Authorization: token {token}`

### PR DESCRIPTION
**Pull request by zeripath**
This PR adds a description to the authorization token header in swagger.

Fix #7582

### CODE REVIEW

File: routers/api/v1/api.go [hunk 1]
```diff
@@ -43,6 +43,7 @@
 //          type: apiKey
 //          name: Authorization
 //          in: header
+//          description: API tokens must be prepended with "token ".
```
Comment (lafriks) [inline routers/api/v1/api.go]:
```suggestion
//          description: API tokens must be prepended with "token".
```

Comment (zeripath) [reply routers/api/v1/api.go]: The space is important - this way it's clear that it has to have "token" and a space there.

**Review by lunny — APPROVED** [review]

File: routers/api/v1/api.go (hunk 1 above)
Comment (zeripath) [reply routers/api/v1/api.go]: How about:

```suggestion
//          description: API tokens must be prepended with "token" followed by a space.
```

**Review by sapk — APPROVED** [review]

File: templates/swagger/v1_json.tmpl [hunk 2]
```diff
@@ -10390,6 +10390,7 @@
       "in": "query"
     },
     "AuthorizationHeaderToken": {
+      "description": "API tokens must be prepended with \"token \".",
```
Comment (zeripath) [inline templates/swagger/v1_json.tmpl]:
```suggestion
      "description": "API tokens must be prepended with \"token\" followed by a space.",
```

**Review by sapk — APPROVED** [review]

**Pull request merged**